app/templates/home-view.jade

```diff
@@ -1,331 +1,547 @@
-                  if total === 0
-                    span= course.get('description')
-              img.media-object(src="/images/pages/home/" + course.get('slug') + ".png")
-        .col-md-3.col-sm-4
-          .media.disabled
-            .media-body
-              p.small(data-i18n="new_home.coming_soon")
-            img.media-object(src="/images/pages/home/inprogress.png")
-
-        .clearfix
-        .text-center
-          h4
-            img(src="/images/pages/about/new_languages.png")
-            div(data-i18n="new_home.courses_available_in")
-
-    .testimonials-rows
-      .testimonials-filler-left
-      .testimonials-filler-right
-      .row
-        .col-lg-offset-2.col-lg-7.col-sm-8
-          blockquote
-            h3(data-i18n="new_home.boast")
-
-        .col-lg-2.col-sm-3.text-center
-          img.img-circle(src="/images/pages/home/opensource.png")
-          h6 Open Source
-          .small opensource.com
-
-      .row
-        .col-lg-7.col-sm-8.col-sm-push-4.col-lg-push-3
-          blockquote
-            h3(data-i18n="new_home.winning")
-
-        .col-lg-2.col-sm-3.col-lg-offset-1.text-center.col-sm-pull-8.col-lg-pull-7
-          img.img-circle(src="/images/pages/home/pcmag.png")
-          h6 PC Mag
-          .small pcmag.com
-
-    .request-demo-row.text-center
-      h3(data-i18n="new_home.run_class")
-      if view.isTeacherWithDemo
-        div
-          button.btn.btn-primary.btn-lg.setup-class-btn(data-event-action="Homepage Setup Class Page Bottom", data-i18n="new_home.setup_a_class")
-      else
-        div
-          button.btn.btn-primary.btn-lg.request-demo(data-event-action="Homepage Request Demo Page Bottom", data-i18n="new_home.request_demo")
+          .text-center
+            button.btn.btn-lg.btn-primary.request-demo(data-i18n="new_home.request_quote")
+
+    .row.row-dark.text-center#built-for-teachers
+      .col-lg-12
+        h1.text-gold(data-i18n="new_home.built_for_teachers_title")
+        img(src="/images/pages/home/pink_rectangle.png")
+        .container
+          .row
+            .col-lg-12
+                p(data-i18n="new_home.built_for_teachers_blurb")
+                #built-for-teachers-carousel.carousel.text-left(data-interval=3000)
+                  ol.carousel-indicators
+                    li(data-target="#built-for-teachers-carousel", data-slide-to="0").active
+                    li(data-target="#built-for-teachers-carousel", data-slide-to="1")
+                    li(data-target="#built-for-teachers-carousel", data-slide-to="2")
+                  .carousel-inner
+                    .item.active
+                      .row.hidden-md.hidden-sm.hidden-xs
+                        .col-lg-6
+                          img.computer(src="/images/pages/home/built_for_teachers1.png")
+                        .col-lg-4
+                          h3.text-teal(data-i18n="new_home.built_for_teachers_subtitle1")
+                          p
```

Comment:
This `p` tag doesn't wrap the div beneath it. Is this intentional?
The construct that results is:

```
<p></p>
<div class="text-navy">{raw text}</div>
```

Reply:
Weird, no I had intended to have the `p` wrap.  I guess it could be removed if it's not being used anyway.